Sources/Core/Model/ReactionType.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  ReactionType.swift
+//  StreamChatCore
+//
+//  Created by Alexey Bukhtin on 04/03/2020.
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// A reaction type.
+public enum ReactionType: Codable {
```

Comment:
Can't this be under `Reaction`? so it'd be `Reaction.Type`?

Reply:
It will be required to be:
```swift
Reaction.`Type`
```